css/styles.css

```diff
@@ -20,6 +20,15 @@ li > p {
   z-index: 10 !important;
 }
 
+.skiplink {
+  position: absolute;
+  transform: translateY(-300%);
+}
+
+.skiplink:focus {
+  transform: translateY(0);
+}
```

Comment:
You should use `tailwind` classes :)

Reply:
Yeah, initially wanted to hold back because I'm not vast with `tailwind`, but I'll pick it up and learn so I can make more contributions to the website.